pkg/backend/display/progress.go

```diff
@@ -1390,8 +1390,8 @@ func (display *ProgressDisplay) getStepOp(step engine.StepEventMetadata) deploy.
 	return op
 }
 
-func (display *ProgressDisplay) getStepOpLabel(step engine.StepEventMetadata) string {
-	return display.getStepOp(step).Prefix() + colors.Reset
+func (display *ProgressDisplay) getStepOpLabel(step engine.StepEventMetadata, done bool) string {
+	return display.getStepOp(step).Prefix(done /* color done */) + colors.Reset
```

Comment:
Nit: No need when you're not passing through a true/false
```suggestion
	return display.getStepOp(step).Prefix(done) + colors.Reset
```